src/actions/console.js

```diff
@@ -24,6 +24,14 @@ export const clearConsoleEntries = createAction(
   'CLEAR_CONSOLE_ENTRIES',
 );
 
+export const setPreviousHistoryIndex = createAction(
```

Comment:
I opted for 2 actions each with a single argument, but this could also have been a single `historyPagingState` object with index and current (temporary) input